### ISSUE TITLE
How to Unbind

### ISSUE THREAD

**Issue by antoniobrandao** [tholman/zenpen#107]
How to unbind / clear event listeners? Is this implemented?

**Comment by tholman** [comment]
This isn't implemented... for the most part because the app doesn't have a point where it needs to unbind :)

Wouldn't be too difficult to put it in, but its probably best that you fork it, and do it there... since it doesn't exactly belong in the main app itself :)

**Comment by antoniobrandao** [comment]
I understand the app www.zenpen.io never unbinds... But as a library available to other developers, there are a lot of different scenarios where unbinding is important if not essential.

But take a single-page app as an example. I am in the process of finalising our new backbone-based CMS. It behaves a little bit like Inbox by Google. There is a list of items, if I click one, it expands, the others contract (but remain there) - the clicked item reveals the text editor. I edit the text, click SAVE, then click another item in the same page and edit it's text. If I can't unbind the first one, it will be listening to events for nothing. It's easy to create big memory leaks in single-page-apps when event unbinding isn't carried out appropriately - you're probably well aware.

For this reason we ended up using Pen.js for the text editors in our CMS - a similar text editor, but with the unbind feature. The famous Medium-Editor clone and medium.js also have unbind / deactivate methods, but they were to "big" for us.

Zenpen is more lightweight and looks good, we only moved to pen.js because of the missing unbind feature. The unbind feature would make zenpen.js an option for many other projects! :)

Thanks for your answer !
